app/src/main/java/com/toyou/toyouandroid/presentation/fragment/home/HomeViewModel.kt

```
@@ -60,54 +69,24 @@ class HomeViewModel(private val tokenManager: TokenManager) : ViewModel() {
+                val response = repository.getYesterdayCard()
+                Timber.tag("HomeViewModel").d("yesterdayCards: ${response.result}")
+                if (response.isSuccess) {
+                    _yesterdayCards.value = response.result.yesterday
+                    Timber.tag("HomeViewModel").d("yesterdayCards: ${response.result.yesterday}")
+                } else {
+                    tokenManager.refreshToken(
+                        onSuccess = { getYesterdayCard() },
+                        onFailure = { Timber.tag("HomeViewModel").d("refresh error") }
+                    )
+                }
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)